scripts/sync.py

```diff
@@ -130,7 +130,7 @@ def do(args):
         return True  # break the do function here for the receiving side
 
     # initialize (overwrite) metadata.csv using either REDCap or RPMS database
-    if 'redcap' in args.source or 'rpms' in args.source:
+    if 'redcap' in args.input_sources or 'rpms' in args.input_sources:
         lochness.initialize_metadata(Lochness, args)
```

Comment:
Does `initialize_metadata()` do this parsing (Ln 95 above) again?

> args.source = [SOURCES[x] for x in args.source]

Reply:
inside `initialize_metadata()` method, `intialize_metadata` from `lochness.redcap` and `lochness.rpms`  gets applied accordingly depending on the source (`str`)

The parsing below is not repeated in the `initialize_metadata()` method.
> args.source = [SOURCES[x] for x in args.source]